### PR TITLE
Add OpenClonk.app v7.0

### PR DESCRIPTION
**Pull request by bake**

### CODE REVIEW

File: Casks/openclonk.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'openclonk' do
+  version '7.0'
+  sha256 'e81f7ef8dc5a345101a92f81c630f29443049d83b661b18d8a81f15134350ef2'
+
+  url 'http://openclonk.org/builds/release/7.0/openclonk-7.0-x64.app.zip'
+  name 'OpenClonk'
+  homepage 'http://openclonk.org/'
+  license :isc
+
+  app 'openclonk-c54d917-darwin-amd64-clang.app', target: 'OpenClonk.app'
```
Comment (vitorgalvao) [inline Casks/openclonk.rb]: [Not a good case for renaming](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/app.md#target-should-only-be-used-in-select-cases).

Comment (BakeRolls) [inline Casks/openclonk.rb]: The default (`openclonk-c54d917-darwin-amd64-clang.app`) would be ok?

Comment (vitorgalvao) [inline Casks/openclonk.rb]: Yes. Don’t forget the token and file name.

File: Casks/openclonk.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'openclonk' do
+  version '7.0'
+  sha256 'e81f7ef8dc5a345101a92f81c630f29443049d83b661b18d8a81f15134350ef2'
+
+  url 'http://openclonk.org/builds/release/7.0/openclonk-7.0-x64.app.zip'
```
Comment (vitorgalvao) [inline Casks/openclonk.rb]: `url "http://openclonk.org/builds/release/#{version}/openclonk-#{version}-x64.app.zip"`.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in ee2f09da0f0557ed1eb4ba0884a92683b97816bc. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**